app/models/protocol.rb

```diff
@@ -129,6 +129,10 @@ def is_study?
   def virgin_project?
     selected_for_epic == nil
```

Comment:
and here you "== nil".  Consistancy please.  "selected_for_epic.nil?" is much cleaner